scripts/setup_benchmark.py

```
@@ -0,0 +1,81 @@
+            skills += " LGV"
+        if i % 4 == 0:
+            skills += " TTR"
+        crew_data.append((name, role, skills, "42h"))
+
+    c.executemany(
+        "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+        crew_data,
+    )
+
+    # Insert availability for some crew
+    now = datetime.now()
```

Ruff (0.15.6)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)